ComponentKit/Core/CKComponentBoundsAnimationPredicates.h

```diff
@@ -0,0 +1,13 @@
+//
+//  CKComponentBoundsAnimationPredicates.h
+//  ComponentKit
+//
+//  Created by Oliver Rickard on 3/28/17.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+#import <ComponentKit/CKComponentScopeRoot.h>
+
+BOOL CKComponentBoundsAnimationPredicate(id<CKScopedComponent> controller);
```

Comment:
Not to woodshed too much, but, can we reword this function's name?

`CKComponentHasCustomImplementationForBoundsAnimation()`

or something?

small nit: component, not controller.